app/Support/Calendar/Calculator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Copyright (c) 2023 james@firefly-iii.org
```

Comment:
Feel free to replace this with your own name and link to GitHub. As the author, you have the copyright, even when it's released under the AGPLv3.